### ISSUE TITLE
TypeError: Cannot read property 'query' of undefined

### ISSUE THREAD

**Issue by JoshSmith** [code-corps/code-corps-ember#1502]
https://sentry.io/code-corps/code-corps-ember/issues/402222648/

```
TypeError: Cannot read property 'query' of undefined
  at comparableMoment (vendor/ember-cli/app-boot.js:4:1)
  at comparableMoment (moment/index.js:58:1)
  at comparableMoment (moment/index.js:58:1)
  at comparableMoment (moment/index.js:58:1)
  at comparableMoment (moment/index.js:58:1)
...
(13 additional frame(s) were not displayed)
```

**Comment by JoshSmith** [comment]
This happened when trying to type in the task assignment dropdown. There's clearly a bug here.

**Comment by JoshSmith** [comment]
Closed by #1525
